pkg/kubelet/kubelet_pods_test.go

```diff
@@ -292,12 +353,24 @@ fe00::1	ip6-allnodes
 fe00::2	ip6-allrouters
 203.0.113.1	podFoo.domainFoo	podFoo
 fd00::6	podFoo.domainFoo	podFoo
+`,
+		},
+		{
+			hostIPs:        []string{"203.0.113.1", "fd00::6"},
+			hostName:       "podFoo",
+			hostDomainName: "domainFoo",
+			hostAliases:    []v1.HostAlias{},
+			isSupportIPv6:  false,
+			expectedContent: `# Kubernetes-managed hosts file.
```

Comment:
Super minor but if we are touching it, we can make it nicer:

```
	expectedContent: `
		# Kubernetes-managed hosts file.
		127.0.0.1	localhost
		203.0.113.1	podFoo.domainFoo	podFoo
		fd00::6	podFoo.domainFoo	podFoo
		`,
```

and then before the loop:

```
normalize := func(s string) string {
	return dedent.Dedent(strings.Trim(s, "\n"))
}
```

and then in the test loop:

```
assert.Equal(t, normalize(testCase.expectedContent), string(actualContent), "hosts file content not expected")
```